pkgs/gnomod/gnomod.go

```diff
@@ -0,0 +1,210 @@
+package gnomod
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/gnolang/gno/pkgs/gnolang"
+	"github.com/gnolang/gno/pkgs/std"
+	"golang.org/x/mod/modfile"
+	"golang.org/x/mod/module"
+)
+
+var (
+	source        = "test3.gno.land:36657" // "127.0.0.1:26657"
+	queryPathFile = "vm/qfile"
+)
+
+// ReadModFile reads, parses and validates the mod file at gnomod.
+func ReadModFile(absModPath string) (f *File, err error) {
+	data, err := os.ReadFile(absModPath)
+	if err != nil {
+		return nil, fmt.Errorf("reading gno.mod: %s", err)
+	}
+
+	f, err = Parse(absModPath, data)
+	if err != nil {
+		return nil, fmt.Errorf("parsing gno.mod: %s", err)
+	}
+
+	return f, err
+}
+
+func IsModFileExist(absModPath string) bool {
```

Comment:
This is a `IsFileExist`, and makes the API heavier. Please move this in a `util.go` file from the cmd, or try to find an existing one in the codebase that you can reuse.

Reply:
Resolved-by: 2bb60d640061ec350f4fada0f2c485377d13d657